src/sstruct_mv/sstruct_matmult.c

```diff
@@ -1255,3 +1282,212 @@ hypre_SStructMatrixRTtAP( hypre_SStructMatrix  *RT,
 
    return hypre_error_flag;
 }
+
```

Comment:
Should we keep the following methods or move to a separate branch/remove?

Reply:
Agreed. Removing for now.